models/biocro/tests/testthat/test.met2model.R

```diff
@@ -0,0 +1,22 @@
+context("met2model")
+
+outfolder <- tempfile()
+setup(dir.create(outfolder, showWarnings = FALSE))
+teardown(unlink(outfolder, recursive = TRUE))
+
+test_that("Met conversion runs without error", {
+  skip("BIOCRO met2model is currently broken!")
+  nc_path <- system.file("test-data", "CRUNCEP.2000.nc",
+                         package = "PEcAn.utils")
```

Comment:
Yes. Using a common file has 2 advantages: (1) It saves space in the repo, and (2) it is consistent (so, if we change the standard, we only have to update the test input once rather than in like 10 different places).

Since all of these packages depend on `utils`, I think it's a safe assumption that this file will be available for tests.